src/EditorFeatures/CSharpTest2/Recommendations/RecommenderTests.cs

```diff
@@ -88,21 +83,25 @@ private Task CheckResultAsync(string text, int position, bool absent, CSharpPars
                 }
                 else
                 {
-                    var result = (await RecommendKeywordsAsync(position, context)).Select(k => k.Keyword);
+                    var result = (await RecommendKeywordsAsync(position, context)).Single();
                     Assert.NotNull(result);
-                    Assert.Equal(keywordText, result.Single());
+                    Assert.Equal(keywordText, result.Keyword);
+                    if (matchPriority != null)
+                    {
+                        Assert.Equal(matchPriority.Value, (int)result.MatchPriority);
```

Comment:
The MatchPriority enum is internal to Roslyn and the tests see it through IVT. The test methods are public, so you get an error about the accessibility not matching. The old tests have this same problem with Roslyn's internal Glyph type.